insolar/message/logicrunner.go

```diff
@@ -270,6 +270,39 @@ func (pf *PendingFinished) Type() insolar.MessageType {
 	return insolar.TypePendingFinished
 }
 
+// AdditionalCallFromPreviousExecutor is sent to the current executor
+// by previous executor when Flow cancels after registering the request
+// but before adding the request to the execution queue. For this reason
+// this one request may be invisible by OnPulse handler. See HandleCall
+// for more details.
+type AdditionalCallFromPreviousExecutor struct {
```

Comment:
there can be the same problem as was with Pending Requests - who waits for result?

Reply:
Ruslan says it's already handled somewhere in RegisterResults, I don't have to bother about it in this PR.